CHANGELOG.md

```diff
@@ -34,90 +43,6 @@
 
 * Open sources support for `sslkey` and `sslcert` in `citus.node_conninfo`
 
-* Introduces `citus_finish_citus_upgrade()` procedure
```

Comment:
It is hard to find the point where I can find the diff

> Replaces citus.hide_shards_from_app_name_prefixes with citus.show_shards_for_app_name_prefixes

So, we should only have `citus.show_shards_for_app_name_prefixes` because the former is from the beta.

Can you check how `hide_shards_from_app_name_prefixes`  was added, and use it with `show_shards_for_app_name_prefixes` ?

Reply:
I see that #5567 introduced `hide_shards_from_app_name_prefixes`

The changelog entry for that was `Hides shards based on application_name prefix`.

I guess I can drop the entry for rename, and keep the entry above as is. The name of the GUC changed, but the purpose is still the same.